packages/config/config/devices/0x031e/lzw31-sn.json

```
@@ -367,8 +700,9 @@
 			"allowManualEntry": true
 		},
 		"51": {
-			"label": "Physical On/Off Delay",
-			"description": "The delay that occurs after pressing the physical button to turn the switch on/off is removed. Consequently this also removes the following scenes: 2x, 3x, 4x, 5x tap. 1x tap and config button scenes still work.",
+			"$if": "firmwareVersion >= 1.47",
+			"label": "Instant On",
+			"description": "Controls the 700ms button delay. Note, if you disable the delay, it will also disable multi-tap scenes.",
```

```suggestion
			"description": "Enabling this disables the 700ms button delay and multi-tap scenes.",
```
Seems a bit less confusing to me.